eole/scorers/comet.py

```diff
@@ -0,0 +1,46 @@
+from .scorer import Scorer
+from eole.scorers import register_scorer
+from comet import download_model, load_from_checkpoint
+
+
+@register_scorer(metric=["COMET", "COMET-KIWI", "COMETINHO"])
+class CometScorer(Scorer):
+    """Comet scorer class."""
+
+    def __init__(self, config, model_name):
+        """Download if needed and load Comet model."""
+        super().__init__(config, model_name)
+        self.model_name = model_name
+        if self.model_name == "COMET":
+            comet_model_name = "Unbabel/wmt22-comet-da"
+        elif self.model_name == "COMETINHO":
+            comet_model_name = "Unbabel/eamt22-cometinho-da"
+        elif self.model_name == "COMET-KIWI":
+            comet_model_name = "Unbabel/wmt22-cometkiwi-da"
+        try:
+            comet_model_path = download_model(comet_model_name)
```

Comment:
I don't think the `model_name` attribute is necessary here (and overcrowding all other signatures).
Could we rely on some class based metric name instead?
See how it's done for transforms:
- https://github.com/eole-nlp/eole/blob/4a3d0dd3ba9667d0519d846f68b01a1be5fea94e/eole/transforms/transform.py#L22
- https://github.com/eole-nlp/eole/blob/4a3d0dd3ba9667d0519d846f68b01a1be5fea94e/eole/transforms/__init__.py#L46-L47